tests/TestRunner/app/ApiTests.js

```diff
@@ -432,4 +432,17 @@ describe(module.id, function () {
             expect(NSDate.dateWithTimeIntervalSinceReferenceDate(0).class()).toBe(NSDate);
         });
     }
+    
+    it("should be able to iterate over NSFastEnumeration objects", function () {
```

Comment:
`NSArray` always returns its inner pointerand never uses the buffer provided. What I can do is add another test that uses a big array and iterates over its 'reverseObjectsEnumerator' which does copy items to the buffer provided.